src/doc/en/developer/packaging.rst

```diff
@@ -599,16 +599,31 @@ For example, the ``scipy`` ``spkg-check.in`` file contains the line
 
     exec python3 spkg-check.py
 
+The install-requires.txt file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 All normal Python packages and all wheel packages must have a file ``install-requires.txt``.
```

Comment:
`install-requires.txt` file attains a special position because of the new header. I think this is unbalanced.
 
remove lines 602-604?